### PR TITLE
add Dockerfile

### PR DESCRIPTION
**Pull request by ntphiep**
going to Dockerize this...

### CODE REVIEW

**Comment by ntphiep** [comment]
seem quite ok huh?

**Comment by leoduardo** [comment]
that's ok bro, nice change!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Use an official GCC image as a base image
+FROM gcc:latest
+
+# Set the working directory in the container
```
Comment (ntphiep) [inline Dockerfile:4]: ok

**Review by ntphiep — COMMENTED** [review]
go dockerize

**Review by leoduardo — COMMENTED** [review]
kkk

**Pull request merged**